.github/ci.sh

```diff
@@ -85,9 +85,16 @@ install_llvm() {
 }
 
 install_solvers() {
-  (cd $BIN && curl -o bins.zip -sL "https://github.com/GaloisInc/what4-solvers/releases/download/$SOLVER_PKG_VERSION/$BUILD_TARGET_OS-$BUILD_TARGET_ARCH-bin.zip" && unzip -o bins.zip && rm bins.zip)
-  cp $BIN/yices_smt2$EXT $BIN/yices-smt2$EXT
-  chmod +x $BIN/*
+  snapshot=20250326
+  curl \
+    --fail \
+    --location \
+    --proto '=https' \
+    --show-error \
+    --silent \
+    --tlsv1.2 \
+    "https://github.com/GaloisInc/what4-solvers/blob/d8d6849dd63a4655e1ff07a824c7ec4f234c91a6/scripts/install.sh" | \
+    bash -s -- "${snapshot}" "${BIN}"
```

Comment:
Currently just testing https://github.com/GaloisInc/what4-solvers/pull/66. Will need to be updated to take `SOLVER_PKG_VERSION` before merge. Callsites will need to be modified to stop specifying OS/Arch.